src/Applications/GEOSdas_App/fvsetup

```diff
@@ -5201,7 +5211,7 @@ EOF
        $nx = 24;
        $ny = $nx * 6;
        $km = $vres;
-       $dt = 450;
+       $dt = 300;
```

Comment:
This change of dt (heartbeat?) seems unrelated to LDAS.  Is this intentional?

Reply:
this change was carried from adas5.41.0 ( likely ahead of develop for this modification), it is unrelated to LDAS. will consult with Ricardo and Joe about this.